src/parser/parser_flow.ml

```diff
@@ -105,6 +105,16 @@ let rec filter_duplicate_errors acc = function
     filter_duplicate_errors acc rl
 | x :: rl -> filter_duplicate_errors (x :: acc) rl
 
+let unexpected_error = function
```

Comment:
Haha. I was definitely being a bit cute here, but I can see how this is confusing. `get_unexpected_error` works for me.